### ISSUE TITLE
Strange Axis labels

### ISSUE THREAD

**Issue by forki** [fsharp/FSharp.Charting#63]
I noticed that FSharp.Charting is creating strange Axis labels:

![image](https://cloud.githubusercontent.com/assets/57396/5661664/aaa16444-972e-11e4-9217-ba0e8184aa69.png)

this is how Octave does it:

![image](https://cloud.githubusercontent.com/assets/57396/5661652/87875d92-972e-11e4-9fd2-048b17103a36.png)

**Comment by forki** [comment]
It's also strange that X-Axis starts at 30, but y-Axis starts at 0.

**Comment by dsyme** [comment]
Agreed - there must surely be some default charting setting to do a better job on this.  If you can find the appropriate setting, we can make it the default.
